src/Network/Network/help/New-AzVirtualNetworkSubnetConfig.md

```diff
@@ -68,7 +69,7 @@ This example creates two new subnet configurations using the
 Specifies a range of IP addresses for a subnet configuration.
 
 ```yaml
-Type: System.String[]
+Type: String[]
```

Comment:
When you do `Update-MarkdownHelpModule` use also the parameter `-UseFullTypeName`. This would keep the use of full type names.

Reply:
updated, please review